Procfile.learner

```diff
@@ -0,0 +1,18 @@
+# Use goreman to run `go get github.com/mattn/goreman`
+
+# 1. Start the cluster uring Procfile
+# 2. Add learner node to running cluster
+#   % etcdctl member add infra4 --peer-urls="http://127.0.0.1:42380" --learner=true
+#   Member 4bd62934a5391059 added to cluster ef37ad9dc622a7c4
+
+#   ETCD_NAME="infra4"
+#   ETCD_INITIAL_CLUSTER="infra4=http://127.0.0.1:42380,infra1=http://127.0.0.1:12380,infra2=http://127.0.0.1:22380,infra3=http://127.0.0.1:32380"
+#   ETCD_INITIAL_ADVERTISE_PEER_URLS="http://127.0.0.1:42380"
+#   ETCD_INITIAL_CLUSTER_STATE="existing"
+
+# 3. export the above environment variables
+# 4. start learner node with goreman
+etcd4: bin/etcd --listen-client-urls http://127.0.0.1:42379 --advertise-client-urls http://127.0.0.1:42379 --listen-peer-urls http://127.0.0.1:42380 --initial-cluster-token etcd-cluster-1 --enable-pprof --logger=zap --log-outputs=stderr 
+
+
```

Comment:
Remove extra empty lines?

Reply:
@vimalk78 thanks so much for addressing all the comments. We should still remove one extra line here (LOC 9 or 10 :))

Reply:
done.